packages/headless/docs.config.json

```diff
@@ -141,6 +141,27 @@
     ]
   },
   "controllers": [
+    {
```

Comment:
Is it good? How do I test that it works?

Reply:
This file is no longer being used, it was part of the original approach to documentation. Only the `doc-parser.ts` changes are needed.

To test, run:
1. `npm run build`
2. `npm run doc:generate`
3. Inspect the `parsed_doc` file inside the `dist/` directory.

I created [KIT-693](https://coveord.atlassian.net/browse/KIT-693) to clean up the old approach.